BookPlayer/Utils/Extensions/AppHostingViewController.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  AppHostingViewController.swift
+//  BookPlayer
+//
+//  Created by Gianni Carlo on 13/8/25.
+//  Copyright © 2025 BookPlayer LLC. All rights reserved.
+//
+
+import BookPlayerKit
+import SwiftUI
+
+final class AppHostingViewController<Content: View>: UIHostingController<Content> {
+  override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
```

Comment:
Do you need this if you already have it part of the `AppNavigationController`?

Reply:
yes, the `AppNavigationController` will eventually be deleted when we migrate the player screens to SwiftUI, and the only way to force the orientation lock is via this bridge with UIKit which hosts the `MainView`, the options available in SwiftUI were not good enough :/